src/nuclide.cpp

```diff
@@ -191,6 +193,47 @@ Nuclide::Nuclide(hid_t group, const double* temperature, int n)
   }
   close_group(rxs_group);
 
+  // Read unresolved resonance probability tables if present
+  if (object_exists(group, "urr")) {
+    urr_present_ = true;
+    urr_data_.resize(temps_to_read.size());
```

Comment:
If you use `emplace_back` below, this call should be `urr_data_.reserve(...)` instead of `resize`